package.json

```diff
@@ -14,8 +14,8 @@
     "test:node": "jest",
     "test:optimizer:snapshot": "OPTIMIZER_SNAPSHOT=true jest",
     "test:browser": "karma start --single-run --browsers ChromeHeadless karma.conf.js",
-    "lint": "eslint  -c .eslintrc.json . && lerna run lint",
-    "lint:fix": "eslint  -c .eslintrc.json --fix . && lerna run lint:fix"
+    "lint": "eslint  -c .eslintrc.json .",
+    "lint:fix": "eslint  -c .eslintrc.json --fix '**/*.{js,ts}'"
```

Comment:
Is it ensured that the fix is only run in the source folders and not node_modules and dist folders?

Reply:
We now run eslint from the root dir for all projects. Before linter had it's own eslint config.